src/components/search/CragFinder.tsx

```diff
@@ -17,35 +17,28 @@ export interface CragFinderProps {
 
 const CragFinder = ({ isMobile = true, placeholder = 'Try \'Smith Rock\', \'Las Vegas\'' }: CragFinderProps): JSX.Element => {
   const router = useRouter()
-  useEffect(() => {
-    if (isMobile) return
-    const inputs = document.getElementsByClassName('aa-Input')
-    for (let i = 0; i < inputs.length; i++) {
-      (inputs[i] as HTMLElement).focus()
-    }
-  })
+
   return (
     <Autocomplete
       id='crag-finder'
       classNames={{ item: 'crag-finder-item', panelLayout: 'crag-finder-panelLayout' }}
       placeholder={placeholder}
-      getSources={async ({ query }) => {
+      getSources={({ query }) => {
         if ((query as string).length < 3) {
```

Comment:
@vnugent this check to prevent running queries (not even a debounced query) exists in the CragFinder component, but not in ClimbSearchByName component. Should these components match in this way or leave as is?

Reply:
sure, let's add it to search by name as well.

Reply:
Removed, should be ready for review now.